.ci/load/Jenkinsfile

```diff
@@ -0,0 +1,169 @@
+// For documentation on this pipeline, please see the README.md in this directory
+pipeline {
+    agent any
+    environment {
+        REPO = 'apm-agent-java'
+        APP = 'spring-petclinic'
+        APP_BASE_DIR = "src/${APP}"
+        METRICS_BASE_DIR="metrics/"
+        AGENT_BASE_DIR = "agent/"
+        ORCH_URL = 'obs-load-orch.app.elastic.co:8000'
+        // Set below for local development
+        // ORCH_URL='10.0.2.2:8000'
+        DEBUG_MODE = '0' // set to '0' for production
+        // This is a placeholder server. This will change to a dummy APM Server when it is available.
+        APM_SERVER_URL = 'https://2a2bd0e2806a47e5996eeeec6d22e6df.apm.eu-west-3.aws.elastic-cloud.com:443'
+        LOCUST_RUN_TIME = "${params.duration}"
+        LOCUST_USERS = "${params.concurrent_requests}"
+
+    }
+    options {
+        timeout(time: 3, unit: 'HOURS')
```

Comment:
[minor] for long running stability tests, we might want to have longer or no timeouts

Reply:
How about I set this to 72 hours for now which should be more than long enough and then we can re-visit it if we find a need for multi-day tests? Let me know if that sounds all right.

Reply:
sgtm